### ISSUE TITLE
Enabling Water script causes screen to go black (2017.3.0f3)

### ISSUE THREAD

**Issue by mold** [Moonlit-Games/Ultimate-Water-System#40]
I upgraded to the latest version by installing 1.0.1 from Asset store and then applying hotfix b3 from http://goo.gl/nBx7dS. This is the only way to update that I could find, even though going from 1.0 to 2.0 via a hotfix seems weird.

When I enable the Water gameobject is enabled in the scene, the game display goes completely black. I don't know if it's related, but I also get this error in the console:

```
Shader error in 'UltimateWater/Variations/Water _ALPHABLEND_ON _CUBEMAP_REFLECTIONS _INCLUDE_SLOPE_VARIANCE _PLANAR_REFLECTIONS _WATER_RECEIVE_SHADOWS _WATER_REFRACTION _WAVES_FFT': Fragment program 'fragForwardBase': Struct variable 'globalWaterData' is ignored. Only instancing constant buffers can have struct variables
 (on d3d11)

Compiling Fragment program with DIRECTIONAL LIGHTPROBE_SH UNITY_SINGLE_PASS_STEREO _WAVES_FFT _WATER_BACK
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_RGBM_ENCODING
```

It worked with 1.0.0 in 2017.1.1 before upgrading to 2017.3.0f3 and 2.0.0b3

**Comment by MCPGNZ** [comment]
Hi, this problem is a duplicate of [this issue](https://github.com/Moonlit-Games/Ultimate-Water-System/issues/38).

It's already solved in the UWS 2.0.1, (sent to asset store and it's waiting for approval) 
as it can take a while for Unity team to push the release, you can fix it using the solution provided in the link above.

**Comment by mold** [comment]
Ok, thanks!
